.github/workflows/linting.yml

```
@@ -13,6 +13,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Run codegen
         run: make codegen
+      - name: Make sure there's no pending changes
```

```suggestion
      - name: Verify generated code matches committed code
```

Sorry for the nit, started with avoiding "there's"->"there are".